ocw/lib/db.py

```diff
@@ -19,157 +18,102 @@
-
-
-def ec2_to_local_instance(instance, vault_namespace, region):
-    csp_info = {
-        'type': instance.instance_type,
-        'launch_time': instance.launch_time.isoformat(),
-        'tags': {t['Key']: t['Value'] for t in instance.tags} if instance.tags else {}
+        csp_info = CspInfo(tags=csp_data['tags'], type=csp_data['type'], instance=local_instance)
+        csp_info.save()
+    local_instance.set_alive()
+    local_instance.save()
+
+def ec2_extract_data(csp_instance, namespace, region, default_ttl):
+    return {
+        'tags' : {t['Key']: t['Value'] for t in csp_instance.tags} if csp_instance.tags else {},
+        'id' : csp_instance.instance_id,
+        'first_seen' : dateutil.parser.parse(csp_instance.launch_time.isoformat()),
+        'namespace': namespace,
+        'region': region,
+        'provider': ProviderChoice.EC2,
+        'type': csp_instance.instance_type,
+        'default_ttl': default_ttl
     }
-    return Instance(
-        provider=ProviderChoice.EC2,
-        vault_namespace=vault_namespace,
-        first_seen=dateutil.parser.parse(csp_info.get('launch_time', str(timezone.now()))),
-        instance_id=instance.instance_id,
-        state=StateChoice.ACTIVE,
-        region=region,
-        csp_info=json.dumps(csp_info, ensure_ascii=False),
-        ttl=timedelta(seconds=int(csp_info['tags'].get(
-            'openqa_ttl', PCWConfig.get_feature_property('updaterun', 'default_ttl', vault_namespace)))),
-        ignore=tag_to_boolean('pcw_ignore', csp_info)
-    )
 
 
-def azure_to_json(i):
-    info = {'tags': i.tags if i.tags else {}}
-    if (i.tags is not None and 'openqa_created_date' in i.tags):
-        info['launch_time'] = i.tags.get('openqa_created_date')
-    return info
-
-
-def azure_to_local_instance(instance, vault_namespace):
-    csp_info = azure_to_json(instance)
-    return Instance(
-        provider=ProviderChoice.AZURE,
-        vault_namespace=vault_namespace,
-        first_seen=dateutil.parser.parse(csp_info.get('launch_time', str(timezone.now()))),
-        instance_id=instance.name,
-        region=instance.location,
-        csp_info=json.dumps(csp_info, ensure_ascii=False),
-        ttl=timedelta(seconds=int(csp_info['tags'].get(
-            'openqa_ttl', PCWConfig.get_feature_property('updaterun', 'default_ttl', vault_namespace)))),
-        ignore=tag_to_boolean('pcw_ignore', csp_info)
-    )
-
-
-def gce_to_json(i):
-    info = {
-        'tags': {m['key']: m['value'] for m in i['metadata']['items']} if 'items' in i['metadata'] else {},
-        'type': GCE.url_to_name(i['machineType']),
-        'launch_time': str(i['creationTimestamp'])
+def azure_extract_data(csp_instance, namespace, default_ttl):
+    tags = csp_instance.tags if csp_instance.tags else {}
+    first_seen = dateutil.parser.parse(tags.get('openqa_created_date', str(timezone.now())))
+    return {
+        'tags' : tags,
+        'id' : csp_instance.name,
+        'first_seen' : first_seen,
```

Comment:
```suggestion
        'tags': tags,
        'id': csp_instance.name,
        'first_seen': first_seen,
```
Pylint may complain about whitespace before ':'